Sources/Shared/UI/Documents/DocumentsViewController.m

```diff
@@ -1068,15 +1077,16 @@ - (void)imagePickerController:(UIImagePickerController *)picker didFinishPicking
                 
                 //release the date formatter because, not needed after that piece of code
                 [dateFormatter release];
-                imageName = [NSString stringWithFormat:@"MobileImage_%@.png", tmp];
+                imageName = [NSString stringWithFormat:@"mobile_image_%@.png", tmp];
```

Comment:
Like in extension, we have the same regles of file name to use eCMS to upload, hasn't it?